roles/dnsmasq/defaults/main.yml

```
@@ -19,3 +19,9 @@
 
 cifmw_dnsmasq_basedir: "/etc/cifmw-dnsmasq.d"
 cifmw_dnsmasq_global_options: {}
+cifmw_dnsmasq_forwarders: []
+cifmw_dnsmasq_interfaces: []
+cifmw_dnsmasq_listen_addresses: []
+cifmw_dnsmasq_enable_dns: false
+cifmw_dnsmasq_addresses: []
+cifmw_dnsmasq_domainname: ''
```

this ready better maybe?
```suggestion
cifmw_dnsmasq_domain_name: ''
```